src/index.css

```
@@ -0,0 +1,8 @@
+html {
+    height: 100vh;
+    width: 100%;
+}
+
+body {
+    margin: 0;
+}
```

```suggestion
}

```

As preferred by Dr. @blauks